deploy/examples/csi/rbd/storageclass-ec.yaml

```diff
@@ -59,6 +59,16 @@ parameters:
   # https://docs.ceph.com/docs/master/man/8/rbd-nbd/#options
   # unmapOptions: force
 
+   # (optional) Instruct the plugin it has to encrypt the volume
+   # By default it is disabled. Valid values are "true" or "false".
+   # A string is expected here, i.e. "true", not true.
+   # encrypted: "true"
```

Comment:
Let's not add the encryption to all the storage class yaml examples. Let's just add it to one of them to avoid redundancy. I think rbd/storageclass.yaml will be the best place for this. Similarly, we don't add all settings to all examples for the CRs.